### DISCUSSION TITLE
IXFR and AXFR to same Time

### DISCUSSION THREAD

**Discussion by ipv64net** [PowerDNS/pdns#14657] [Q&A]
I have a master slave situation and all domains are transferred properly from master to slave, but always with AXFR.

Now I want to activate IXFR and set the point "set-meta IXFR 1" on the slave.

It activates that, but the master still sends 1x IXFR and 1x AXFR.

Sep 10 18:25:47 nameserver1 pdns_server[115617]: IXFR-out zone 'xxxxx', client '10.10.0.3', transfer initiated with serial 2024090891
Sep 10 18:25:47 nameserver1 pdns_server[115617]: AXFR-out zone 'XXXX', client '10.10.0.3', transfer initiated

If I set disable-axfr, the IXFR no longer works either.

**Comment by omoerbeek** [comment]
At this point in time, the authoritative server does not support IXFR as a server, so the transfer falls back to AXFR. 
Yout can try `ixfrdist`: https://docs.powerdns.com/authoritative/manpages/ixfrdist.1.html

**Comment by klaus-nicat** [comment]
IIRC inbound IXFR can also cause problems (Empty Non-Terminal RRs are not deleted on IXFR)
